### DISCUSSION TITLE
Add a trigger that gets which helper slot the helper is filling

### DISCUSSION THREAD

**Discussion by ghost** [ikemen-engine/Ikemen-GO#1060] [Feature Request]
As I am developing a clone army type of character/minigame, I want to be able to display their health meters stack like this (apologizes for the ascii art thing):
||||||
||||||||||||
|||||||||
When a enemy dies that was inbetween the first and last, that spot remains empty until another clone fills that spot in.

### Describe the solution you'd like

Unfortunately, Ikemen GO does not have a single trigger that makes this significantly easier to manage, I have to devise complicated code in order to do this.

Possible names for the trigger: helperindex, helperslot, playerslot

What it will do is get where they are in the player slots, starting from 4,5,6,7,etc. depending on how many non-helper players there are in the fight, starts at 3 if there are only two.

Or you can make it always start at 1 for the first helper slot filled, 2, 3, and so on.

**Comment by potsmugen** [comment]
The helper index "trigger" already exists in the source code so it would be easy to implement, I think. Could be useful.
